near-contract-standards/src/fungible_token/core_impl.rs

```diff
@@ -133,17 +133,20 @@ impl FungibleTokenCore for FungibleToken {
         msg: String,
     ) -> PromiseOrValue<U128> {
         assert_one_yocto();
-        require!(env::prepaid_gas() > GAS_FOR_FT_TRANSFER_CALL, "More gas is required");
+        require!(
+            env::prepaid_gas().as_gas() > GAS_FOR_FT_TRANSFER_CALL.as_gas(),
```

Comment:
As you already implemented Ord and PartialOrd for NearGas in your PR, you can revert this change back to:

```rust
env::prepaid_gas() > GAS_FOR_FT_TRANSFER_CALL,
```